src/SpikeCore/SpikeCore/SpikeCore.csproj

```diff
@@ -7,13 +7,15 @@
 
   <ItemGroup>
     <PackageReference Include="Foundatio" Version="7.1.1845" />
+    <PackageReference Include="Json.Net" Version="1.0.16" />
```

Comment:
A `Newtonsoft.Json` counterfeit? That package does not provide `using Newtonsoft.Json.Linq;`, that namespace is only available because an ASP.NET Core package is referencing it and it is available to you because SDK style projects reference all transitively referenced references.

Reply:
Yeah, I thought that version number seemed pretty damn low too. I wonder where the hell I got the good idea that was the right package... probably one of the NuGet UIs, and not paying a lot of attention. In my defense, `Kingdom` is a really great show, and it only has subtitles you see... ;)